compiler/rustc_hir_typeck/src/method/suggest.rs

```diff
@@ -1411,6 +1435,7 @@ impl<'a, 'tcx> FnCtxt<'a, 'tcx> {
                     ProbeScope::TraitsInScope,
                 ) {
                     // found a match, push to stack
+                    // FIXME: should check all impl candidates, see issue#124946
```

Comment:
This comment confuses me. If somebody in the future reads this, they will go to issue #124946 and see that it was fixed by #125080, and there doesn't seem to be any explanation relating to impl candidates. Perhaps expand the comment to include more detail of what needs fixing?